components/participant_table/commons/participant_table_base.lua

```diff
@@ -354,8 +360,10 @@ end
 ---@param section ParticipantTableSection
 ---@param amountOfEntries number
 ---@return Html?
-function ParticipantTable.sectionTitle(section, amountOfEntries)
-	if Logic.isEmpty(section.config.title) then return end
+function ParticipantTable:sectionTitle(section, amountOfEntries)
+	if Logic.isEmpty(section.config.title) or section.config.title == self.config.title then
+		return
+	end
```

Comment:
> suppress duplicate title when title is set in participant table and there are no sections (i.e. the whole table is treated as a section)

This is not entirely correct, as this will also suppress section titles when they are the same as the table title,
so something like

```
Participants
Invited
Participants
```
would omit the second `Participants`. I don't know whether a scenario like is exists or will exist in the future.
I thought about modifying the args for single-section tables, but that might require copying them (not sure how and when the args object is used), so solved the issue this way.

Reply:
hmm, true, but i think it will not happen and if it does we can adjust then imo

Reply:
Yeah, that's why i wrote this solution, i just wanted to mention this to other reviewers